tests/src/Discovery/PathFinderBaseTest.php

```diff
@@ -49,7 +75,7 @@ public function cleanDirPathProvider()
 class FakePathFinder extends PathFinderBase {
 
   public function find($seed) {
-
+    return realpath($seed);
```

Comment:
Maybe something like:

```php
return $this->path;
```

Actually, that is exactly what `PathFinderNull` does, so maybe we can use that class.